testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -557,6 +559,18 @@ def test_Cluster_add_metadata(self, cluster):
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+  
+    def test_empty_Cluster(self):
+        empty_cluster = encore.Cluster()
+        assert empty_cluster.size == 0 
+        assert np.size(empty_cluster.elements) == 0 
+        assert empty_cluster.centroid is None
+        assert bool(empty_cluster.metadata) == False
+      
+    def test_centroid_not_in_elements(self):
+        with pytest.raises(LookupError):
```

Comment:
Can you match on some of the text being emitted with the LookupError here please? That can be very useful in checking that you're triggering the right error with a test.

Reply:
That's a good point that I didn't cover, I'm working on it

Reply:
done ✔️